collector/exporter/otelarrowexporter/README.md

```diff
@@ -54,7 +86,45 @@ Several helper files are leveraged to provide additional capabilities automatica
 - [TLS and mTLS settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md)
 - [Queuing, retry and timeout settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md)
 
-[beta]: https://github.com/open-telemetry/opentelemetry-collector#beta
-[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
-[core]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol
-[stable]: https://github.com/open-telemetry/opentelemetry-collector#stable
+### Arrow-specific Configuration
+
+In the `arrow` configuration block, the following settings enable and
+disable the use of OTel Arrow as opposed to standard OTLP.
+
+- `disabled`: disables use of Arrow, causing the exporter to use standard OTLP
+- `disable_downgrade`: prevents this exporter from using standard OTLP
+
+The following settings determine the resources that the exporter will use:
+
+- `num_streams`: the number of concurrent Arrow streams
+- `max_stream_lifetime`: duration after which streams are recycled
+
+### Experimental Configuration
+
+The exporter supports configuring compression at the Arrow IPC level.
+
+- `payload_compression`: compression applied at the Arrow IPC level, "none" by default, "zstd" supported.
+
+The exporter uses the signal-specific Arrow stream methods (i.e.,
+`ArrowTraces`, `ArrowLogs`, and `ArrowMetrics`) by default.  There is
+an option to use the generic `ArrowStream` method instead.
```

Comment:
Out of curiosity - what situations would a user want to use ArrowStream method over the per-signal method?

Reply:
I had a few things in mind -- the original proto had just one stream, but we wanted to configure Envoy for signal-specific arrow stream destinations, so I switched and left the option for experiments.

The benefit, hypothetically, would be that traces/metrics/logs from a single source share tons of common attributes, so that traces/metrics/logs will have the same shared-data sections, thus more sharing.